apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js

```diff
@@ -0,0 +1,330 @@
+import React from 'react';
+import { assert, expect } from 'chai';
+import { mount } from 'enzyme';
+import TeacherHomepage from '@cdo/apps/templates/studioHomepages/TeacherHomepage';
+
+const announcements = [
+  {
+    heading: "first announcement heading",
+    buttonText: "first announcement buttonText",
+    description: "first announcement description",
+    link: "first announcement link"
+  },
+  {
+    heading: "second announcement heading",
+    buttonText: "second announcement buttonText",
+    description: "second announcement description",
+    link: "second announcement link"
+  }
+];
+
+const courses = [
+  {
+    name: "Course 1 name",
+    description: "Course 1 description",
+    link: "Course 1 link",
+    assignedSections: []
+  },
+  {
+    name: "Course 2 name",
+    description: "Course 2 description",
+    link: "Course 2 link",
+    assignedSections: []
+  },
+];
+
+const sections = [
+  {
+    name: "Section 1 name",
+    linkToProgress: "Section 1 link",
+    course: "Section 1 course",
+    linkToCourse: "Section 1 link to course",
+    numberOfStudents: 1,
+    linkToStudents: "Section 1 link to students",
+    sectionCode: "Section 1 section code"
```

Comment:
In my mind, tests serve a couple different purposes. One of those is to to make sure things work like you expect/continue to work like you expect as you make changes.

However, another valuable aspect of tests is to document how you expect them to be used. When I look at some component in code, it can be hard to visualize exactly what sort of data I expect it to be receiving. Unit tests are a great way of showing exactly what sort of inputs I expect. Having more real looking data here can help me better understand the component.

You could potentially still have the benefits of being able to understand what's happening on your single line by doing something like:
```
// You may or may not find naming these variables differently improves clarity
assert.equal(announcementComponent1.heading, announcementsInputs[0].heading);
```